File: orchestrator-swarm/src/test/java/com/sequenceiq/cloudbreak/orchestrator/swarm/containers/MunchausenBootstrapTest.java
package com.sequenceiq.cloudbreak.orchestrator.swarm.containers;

import com.sequenceiq.cloudbreak.orchestrator.OrchestratorBootstrap;

public class MunchausenBootstrapTest extends AbstractContainerBootstrapTest {

    @Override
    public OrchestratorBootstrap getTestInstance() {
        return new MunchausenBootstrap(getMockedDockerClient(), DUMMY_IMAGE, CMD);
    }
}

File: orchestrator-swarm/src/test/java/com/sequenceiq/cloudbreak/orchestrator/swarm/containers/MunchausenBootstrapTest.kt
package com.sequenceiq.cloudbreak.orchestrator.swarm.containers

import com.sequenceiq.cloudbreak.orchestrator.OrchestratorBootstrap

class MunchausenBootstrapTest : AbstractContainerBootstrapTest() {

    override val testInstance: OrchestratorBootstrap
        get() = MunchausenBootstrap(mockedDockerClient, AbstractContainerBootstrapTest.DUMMY_IMAGE, AbstractContainerBootstrapTest.CMD)
}